### ISSUE TITLE
FNIS won't launch

### ISSUE THREAD

**Issue by padamphet** [ModOrganizer2/modorganizer#573]
# The problem
Hi everyone, 
After installing FNIS via MO2  I have run it 1 time, then impossible to run it again . Won't launch via MO2, but  no error message

# Environment
I use MO2 2.1.5.0 version.
FNIS is installed via MO2, but manually (I strictly follow gamer poet video). Path is E/(private SSD only for skyrim)/steamlibrary/steamapps/common/mo2/mods/FNIS behavior

* Mod Organizer Version that exhibits the issue: MO2   2.1.5.0

* Desktop OS/version used to run Mod Organizer: windows 10  version 1803

# Details
If I run FNIS directlyt from the folder (in explorer), I got the Fnis yellow window. So FNIS work good.
When I run it from MO2 the very first time, I get it work. Then never again. I desinstall and reinstall, then the reinstallation will not run FNIS from MO2 even the first time.
When I try to launch, something happen, and boom, MO2 just come back to normal. But have lines written on the bottom of MO2, can see pics here http://image.noelshack.com/fichiers/2018/46/4/1542299578-fnisbug1.jpg

## USVFS
https://gist.github.com/padamphet/4dbbaed5e290a1aa9b58ee46d2a99b14

## MO Interface
https://gist.github.com/padamphet/65190a01463d5f9284f8d577288231ba

I really hope someone will be able to help. I have been on Step forum, then FNIS forum, and finally they
send me here because they assume it is a MO2 related problem.
Ask me if I forget important information for you to investigate further.
Thanks in advance !!!!!
Alex

**Comment by Al12rs** [comment]
Did you delete the files from overwrite or the output mod you placed them in? Not deleting them can trip up fnis.

**Comment by padamphet** [comment]
Hi,

There is no files actually in my overwritten , only FuzRohDU that I should delete each time
________________________________
De : Al <notifications@github.com>
Envoyé : jeudi 15 novembre 2018 17:50
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Did you delete the files from overwrite or the output mod you placed them in? Not deleting them can trip up fnis.

**Comment by padamphet** [comment]
here is a pic http://image.noelshack.com/fichiers/2018/46/4/1542299975-mo2log.jpg
[http://image.noelshack.com/fichiers/2018/46/4/1542299975-mo2log.jpg]
________________________________
De : Al <notifications@github.com>
Envoyé : jeudi 15 novembre 2018 17:50
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Did you delete the files from overwrite or the output mod you placed them in? Not deleting them can trip up fnis.

**Comment by padamphet** [comment]
sorry wrong pic, this is here the pic of overwriiten file http://www.noelshack.com/2018-46-4-1542300859-overwritten.jpg
________________________________
De : Al <notifications@github.com>
Envoyé : jeudi 15 novembre 2018 17:50
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Did you delete the files from overwrite or the output mod you placed them in? Not deleting them can trip up fnis.

**Comment by padamphet** [comment]
help !!!!! :(

**Comment by padamphet** [comment]
You mean this ??
________________________________
De : Al <notifications@github.com>
Envoyé : dimanche 18 novembre 2018 11:28
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Can you show a picture of the FNIS executable setup in the MO2 executable settings?
Also you might want to use discord instead of GitHub for faster support. Sorry if I didn't answer before but I had to prepare for an exam and I didn't really know what to do with this.

**Comment by padamphet** [comment]
I also send you this, a short video of my FNIS installation and attempt to run, so you see the whole proceess, maybe can help. 1minute20sec video. https://www.youtube.com/watch?v=HKFILbP0zmI
[https://www.bing.com/th?id=OVF.igdjYjoRUF2hU86Cfb%2f9HA&pid=Api]<https://www.youtube.com/watch?v=HKFILbP0zmI>

VID20181118114609<https://www.youtube.com/watch?v=HKFILbP0zmI>
VID20181118114609
www.youtube.com


________________________________
De : Al <notifications@github.com>
Envoyé : dimanche 18 novembre 2018 11:28
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Can you show a picture of the FNIS executable setup in the MO2 executable settings?
Also you might want to use discord instead of GitHub for faster support. Sorry if I didn't answer before but I had to prepare for an exam and I didn't really know what to do with this.

**Comment by Al12rs** [comment]
It doesn't seem you are posting any pictures

**Comment by Al12rs** [comment]
Maybe from phone it doesn't work. Well the procedure you did should be correct.

**Comment by Al12rs** [comment]
Do other programs work fine when run from Mo2? 
Could you make a new empty profile and try running a newly installed FNIS there?

**Comment by padamphet** [comment]
I am sorry I am outside playing music right now. In three hours I will be back home and check your messages . I thank you a lot for trying help me.
________________________________
De : Al <notifications@github.com>
Envoyé : dimanche 18 novembre 2018 13:34:47
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Maybe from phone it doesn't work. Well the procedure you did should be correct.

**Comment by padamphet** [comment]
ok first I  send you the pics didn't work 1st time http://www.noelshack.com/2018-46-7-1542560590-setup.jpg
Then all my programs seems runs good (Dyndolod). 
Ok I just create new profile, so every mods are unchecked. I re-install FNIS, add it as executable. BOOM, same, will mot open.
What I do not understand is that the very first time I watch gamerpoet video and do the installation, I had FNIS running good ! Got the yellow windows and I go trough the process explained in the video. Is it possible something remaining somewhere making the programm impossible to run. I don't see where and how, but why should I be able to run it the very first time and then never again despite new installation ? When I re-install FNIS I first remove it from left panel (right click remove), then I go modify executable, click remove . I don't think to forget anything.

**Comment by Al12rs** [comment]
I really don't know either, never heard of such an issue. Mybe try a completely new instance of MO2 (top left button). Also Make sure that the game folder is clean, maybe run the Steam verification. 
Make sure that the game data folder is vanilla.

**Comment by padamphet** [comment]
Maybe goods news. Everytime I try to run FNIS, I can see a new crashdump created in MO2/crashdump folder. I cannot read it, but can send to you. Only problem is here I cannot attached this file. So how can I send to you ?

**Comment by Al12rs** [comment]
Please instead try sending the logs set to debug of the program crashing:
```
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder". Before sending any Logs make sure the Log Level is set to Debug in Settings\Diagnostics, then close MO2 and clean the logs folder, after that you can reopen MO2 and reproduce the problem once done close MO2, before sending or examining the logs as they are finished writing only after the program is closed; after you have finished testing setting the log level back to default is recommended.
You should always send the mo_interface.log file as that one contains the general information of your setup, for the usvfs log, you should upload the latest one if the issue is about programs started through Mo2.
```
https://cdn.discordapp.com/attachments/458597400638914563/458728891620524032/unknown.png

To upload the logs:
```
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
```

**Comment by Al12rs** [comment]
I don't actually have access to my workstation during the week so I can't do much with a crash dump, maybe if some other dev takes interest he can have a look. To upload it you would need to use a separate service such as Mega or Dropbox

**Comment by padamphet** [comment]
Well the log you already have it on the very first message I sent. I have try just now to launch Fnis several time but it doesn't seems to generate new log. But the MO2/crashdump create new folder everytime I run FNIS . 
I am nivice with drop box but I try to share crrashdump here. Hope it works
https://www.dropbox.com/s/8o852377ihcjedp/ModOrganizer.exe-6016.dmp?dl=0
https://www.dropbox.com/s/1ocm5q9o9dow273/ModOrganizer.exe-8224.dmp?dl=0
https://www.dropbox.com/s/cn0vhrxy2eaw3jt/ModOrganizer.exe-13632.dmp?dl=0

**Comment by Al12rs** [comment]
No I need the usvf logs set to debug mode. See the embedded instructions

**Comment by padamphet** [comment]
Done ! I cleaned the log folders before, setup to debug, then run Fnis 1 time. Close Mo2. And there was only 2 logs in the folder, mo log and usvfs log. I post both here 
   https://gist.github.com/padamphet/1aa16a86bab860cdc7899f652536bfa7

Hope it can help !!

**Comment by padamphet** [comment]
> No I need the usvf logs set to debug mode. See the embedded instructions

Hi, I just want to know if I sent you the correct informations and if you think you will be able to find something with thos "debug" ? :)

**Comment by padamphet** [comment]
> No I need the usvf logs set to debug mode. See the embedded instructions

Hi, I don't want to be insistent (bit I am in fact, sorry), but have no news for 3 days now so I would like to know if you can find anything with those usvf I send you or ......if I should really forget to play with Fnis....  snif

**Comment by Al12rs** [comment]
Sorry I don't always have time. I have looked at the log and it appears you have a fnis installed in the data folder?
At least when run it seems to try to open that one.
Can you please make sure that the data folder is completely vanilla, make a new empty profile, delete all FNIS shortcuts, install FNIS in the new profile, add it from the data tab, make sure the exe path is of the folder you just added. Try running it and send the log in case of failure.

Do you perhaps have a program called macType that changes the font to IOS style in all programs? Or some other program like that?

**Comment by padamphet** [comment]
Looks strange. I definitively no have FNIS in data folder, as the only installation I tried is via mo2. And the path shown in EDIT is correct E:\SteamLibrary\steamapps\common\MO2\mods\FNIS Behavior SE\tools\GenerateFNIS_for_Users. Why would it tried to launch it from a unexistant folder....
Well I don't know anything about mactype, and I am pretty sure that I do not use this knid of program. I leave in france in 2 days and I will not have use skyrim for 1 month since I come back, so I will check for tthis in january unfortunatly now. Thanks for your help by the way.

**Comment by Al12rs** [comment]
We released a new MO2 version on discord could you try it out? https://discord.gg/MB3CyMH

**Comment by padamphet** [comment]
Hi, I am actually in France, and I come back to thailand 2 january, so then I will start again to mod skyrim. Now it will be much more faster so I will restart everything again from the begining, and with this new version.
Thanks for the information, I will send news about this when I will be to the FNIS step  installation.
I wish you happy merry christmas 😊
Alex
________________________________
De : Al <notifications@github.com>
Envoyé : mardi 11 décembre 2018 12:38
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


We released a new MO2 version on discord could you try it out? https://discord.gg/MB3CyMH

**Comment by padamphet** [comment]
Well, back home now 😊  Should I install Mod Organizer 2.1.7alpha5 ??

________________________________
De : Al <notifications@github.com>
Envoyé : mardi 11 décembre 2018 12:38
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


We released a new MO2 version on discord could you try it out? https://discord.gg/MB3CyMH

**Comment by Al12rs** [comment]
Yeah, worth a try. Make sure to unblock it

**Comment by padamphet** [comment]
Unblock it ? what do you mean ?

Also I am not sure what is the best archive OR installer ?

________________________________
De : Al <notifications@github.com>
Envoyé : lundi 7 janvier 2019 12:47
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Yeah, worth a try. Make sure to unblock it

**Comment by Al12rs** [comment]
Could we continue this conversation on discord? We have the bots there helping us with explanations etc and responses are faster.

**Comment by padamphet** [comment]
Well I go outside now, then come back in 3 hours. How can I find you on discord ? I will try to talk to you 😊

________________________________
De : Al <notifications@github.com>
Envoyé : lundi 7 janvier 2019 13:21
À : ModOrganizer2/modorganizer
Cc : padamphet; Author
Objet : Re: [ModOrganizer2/modorganizer] FNIS won't launch (#573)


Could we continue this conversation on discord? We have the bots there helping us with explanations etc and responses are faster.

**Comment by Al12rs** [comment]
Just go in #testers channel and ping me by typing @AL#0012 there. You should be able to find me at the very top of the right pane where the server users are liste, in the Lead Devs Role.

**Comment by SS-magic** [comment]
I am having the exact same issue (i should say i did everything the same)
Did te op had a solution?. Sorry for replying an old thread

**Comment by Kelniimir** [comment]
Its probably too late.
Have this issue also, but in newer MO 2.2.1.
Were able to "Explore Virtual Folder" from MO, navigate to FNIS path and launch it to generate behavior.
It still doesn't work by shortcut.

**Comment by Al12rs** [comment]
Closing this for organization purposes, please open a new issue if you still have this problem, or try visiting the Dev discord.

**Comment by TheDuckSCP** [comment]
Well, im three years late im not sure if it's the exact same I have a similar problem but anyone coming across this using MO2 and they cant open FNIS from the shortcut (well the shortcut on the top toolbar) I was able to create an executable shortcut on the dropdown list with my other executables (SKSE, SSEedit, LOOT, etc.) and that works fine. 
![image](https://user-images.githubusercontent.com/87923832/153742394-6a0b298f-d2b8-4429-bd4d-97f534a8a240.png)

**Comment by LostDragonist** [comment]
That's... how you're supposed to do it?

If by "the shortcut on the top toolbar" you mean the "Run FNIS" plugin in the tools menu, then you shouldn't expect a FNIS window to pop up unless something goes wrong. That's kinda the whole point.
